frontend/src/views/Market.vue

```diff
@@ -38,7 +38,8 @@
                 v-visible="allSearchResults && allSearchResults.length > 0"
                 align="center" v-model="currentPage"
                 :total-rows="allListingsAmount"
-                :per-page="activeType === 'weapon' ? weaponShowLimit : characterShowLimit"
+                :per-page="activeType === 'weapon' ? weaponShowLimit :
```

Comment:
this should be delegated to a function, this is too much logic for a template

Reply:
Created nftListingPageSize method for that purpose